app/eventyay/eventyay_common/templates/eventyay_common/events/meetup_create.html

```
@@ -62,7 +63,53 @@ <h1>{% trans "Create a meetup" %}</h1>
-                {% bootstrap_field basics_form.logo_image layout="control" %}
+                <div class="form-group header-image-form-group{% if basics_form.logo_image.errors or basics_form.header_image_preset.errors %} has-error{% endif %}">
+                    <label class="col-md-3 control-label">
+                        {% trans "Header image" %}
+                    </label>
+                    <div class="col-md-9">
+                        <div class="header-image-preview-container">
+                            <div class="header-image-preview-card">
+                                {% if initial_preset_url %}
+                                    <img id="headerImagePreviewImg" class="header-image-preview-img" src="{{ initial_preset_url }}" alt="{% trans 'Header image preview' %}">
+                                {% else %}
+                                    <img id="headerImagePreviewImg" class="header-image-preview-img header-image-hidden" src="" alt="{% trans 'Header image preview' %}">
```

HTMLHint (1.9.2)

[error] The id value [ headerImagePreviewImg ] must be unique.

(id-unique)